### PR TITLE
2nd assignment

### PR DESCRIPTION
**Pull request by rahulsolanki1812**
@anjanaNewput Please review and merge.

### CODE REVIEW

File: 2nd-assignment/home.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (anjanaNewput) [inline 2nd-assignment/home.html]: Remove extra line

File: 2nd-assignment/home.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            .product {
+                display:  inline-block;
+                width : 33%;
+                background-color : beige;
+                padding : 23px;
+            }
+            .left-category, price {
+                display : inline-block;
+            }
+        </style>
```
Comment (ghost) [inline 2nd-assignment/home.html]: move these styles to css file

File: 2nd-assignment/style.css [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+    width : 100%;
+}
+
+.best-burger > div {
+    color : white;
+    font-size : 70px;
+    font-weight : bold;
+    padding-top :380px;
+}
+.best-burger >div >div {
```
Comment (ghost) [inline 2nd-assignment/style.css]: can we wrap up this in class?

File: 2nd-assignment/style.css [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+    color : white;
+    margin-top :20px;
+    font-size :16px;
+    letter-spacing : 3px;
+    font-weight : 100;
+}
+
+
+/* ARTICLE SECTION */
+.article-div {
```
Comment (ghost) [inline 2nd-assignment/style.css]: rename this, I told for same before

File: 2nd-assignment/style.css [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+    line-height: 30px;
+    margin: 0 auto 20px auto;
+    width: 80%;
+    
+}
+article {
```
Comment (ghost) [inline 2nd-assignment/style.css]: don't provide direct css to html tags. Wrap in some class.

File: 2nd-assignment/style.css [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+    margin-bottom: 35px;
+    text-align : center;
+}
+.info {
+    float : left;
+    width : 85%;
+}
+
+.name {
+    color : #666;
```
Comment (ghost) [inline 2nd-assignment/style.css]: use 6 digit for haxcode

File: 2nd-assignment/style.css [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+}
+
+.black-menu-img {
+    display : block;
+    margin :0 auto;
+    padding-top : 150px;
+    width : 325px;
+}
+
+.p-color {
```
Comment (ghost) [inline 2nd-assignment/style.css]: rename class name.

File: 2nd-assignment/style.css [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+    color :white;
+}
+.p-container {
```
Comment (ghost) [inline 2nd-assignment/style.css]: same here

File: 2nd-assignment/style.css [hunk 9]
```diff
@@ -0,0 +1,358 @@
...
+    }
+    
+    aside {
+        float : none;
+        width : 87%;
+        display : block;
+        margin : 0 auto;
+    }
+    
+    article > h2 {
```
Comment (ghost) [inline 2nd-assignment/style.css]: why using h2? still we are overriding font size? i think no need to use `h2`

File: 2nd-assignment/style.css [hunk 10]
```diff
@@ -0,0 +1,358 @@
...
+        width : 100%;
+    }
+    .reservation-block {
+        height : 480px;
+    }
+    
+    
+}
+
+div: nth-child(3n) {
```
Comment (ghost) [inline 2nd-assignment/style.css]: Is this useful?

**Review by ghost — CHANGES_REQUESTED** [review]
Please fix review comments

**Review by ghost — APPROVED** [review]

**Pull request merged**